src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStore.swift

```diff
@@ -1347,5 +1347,47 @@ extension ContactDiaryStore {
 			migrator: migrator
 		)
 	}
+}
+
+private extension ContactPersonEncounter.Duration {
+
+	var germanDescription: String {
+		   switch self {
+		   case .none:
+			   return ""
+		   case .lessThan15Minutes:
+			   return "< 15 Minuten"
+		   case .moreThan15Minutes:
+			   return "> 15 Minuten"
+		   }
```

Comment:
fixed text? Should we better store them in a separate 'non' localizes string file to make them better find able?

Reply:
This was moved to the ContactDiaryStore file, so right now all the export texts are in here. Could be refactored at some point, but I would like to keep it in this place right now since this affects more than just the texts moved in this task 😁